src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -489,7 +487,58 @@ select game
                 tournamentSсheduleInfo);
         }
 
-        private void ValidateGameInRoundOnCreate(
+        private static void freeDayGameValidation(bool GameValidationIsFreeDayGame, TournamentScheduleDto tournamentScheduleInfo, GameResultDto game)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename method 'freeDayGameValidation' to match camel case naming rules, consider using 'FreeDayGameValidation'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS100)